---
title: "Analyses for paper with Laura and Doris"
author: "Heath Blackmon"
date: "February 1, 2016"
output: html_document
---
We need to add a few analyses to this paper and since we first started a nice tree has become available: [Phylogenomics resolves the timing and pattern of insect evolution](http://science.sciencemag.org.ezp1.lib.umn.edu/content/346/6210/763.full.pdf+html?) this paper includes a consensus tree with node dates equal to the median of from their bayesian analysis.  This tree has 1-4 tips for each order.

```{r}
library(geiger)
library(phytools)
tree <- read.nexus("insects.nex")
plot(tree, cex = .4)
```
```write.csv(tree$tip.label,file="names.csv")```

Outside of R I created a table that has the tree tip names in the first collumn.  The second column has each order name entered only one time (so mostly blanks). So the next step is to prune this tree so we have a single tip for each order.

```{r}
taxa <- read.csv("names.csv", as.is = T)

# create a vector with length the same as number of orders
nums <- rep(1, sum(taxa[, 2] != ""))

# provide the correct order names for the tips we will keep
names(nums) <- taxa[, 1][taxa[, 2] != ""]

# prune the tree to just those tips we are keeping
new.tree <- treedata(tree, nums, warnings=F)[[1]]

# trade out original tip names with order names
for(i in 1:length(new.tree$tip.label)){
  hit <- which(taxa[, 1] == new.tree$tip.label[i])
  new.tree$tip.label[i] <- taxa[hit, 2]
}

# now plot new tree
plot(new.tree, cex = .75)
# that looks good, lets clean up some
rm(taxa, hit, i, nums, tree)

```

Many of our methods are sensitive and perform poorly with trees that have long branches for this reason lets go ahead scale are tree to unit length.  This wont effect relative rates nor transition inference but if we want to report actual rates we will need to untransform.

```{r}

new.tree <-rescale(new.tree, model="depth", 1)
```


Ok now that we have a dated consensus phylogeny we can do several analyses that I felt unsure of previously.  For the first one lets do the reconstruction of the sex chromosome system.  The first problem that we face in this case is that we have variation in some groups however, for most groups there is a clear consensus.  The approach that we will use is to simply take the probability of being in each tip state as being equal to the frequency of that state in an order.  For instance if 90% of records are XY and 10% of records are XO we would place a prior probability of 90% on the state XY and a 10% prior probability on the state XO.

The first thing that we will do is make the empty state matrix
```{r}
scs.states <- c("XY", "XO", "ZW", "ZO", "HD")
states <- matrix(,length(new.tree$tip.label), length(scs.states))
colnames(states) <- scs.states
row.names(states) <- new.tree$tip.label
rm(scs.states)
```

Now we will fill this in
```{r}
setwd("~/Desktop/Dropbox/projects/SexChromosomes (1)/invertebrates/InsectReview/new.version/data")
data <- read.csv("tos.data.old.csv", as.is = T)
for(i in 1:nrow(states)){
  temp.data <- data[tolower(data$Order) == row.names(states)[i], 6]
  states[i, 1] <- sum(temp.data == "XY" | temp.data == "complex XY|homomorphic" | 
                      temp.data == "complex XY" | temp.data == "XY|homomorphic")
  states[i, 2] <- sum(temp.data == "XO" | temp.data == "complex XO")
  states[i, 3] <- sum(temp.data == "ZW" | temp.data == "complex ZW")
  states[i, 4] <- sum(temp.data == "ZO")
  temp.data <- data[tolower(data$Order) == row.names(states)[i], 9]
  states[i, 5] <- sum(temp.data == "paternal genome elimination" | temp.data == "arrhenotoky")
}
# fix notoptera and hemiptera
# hemiptera is based on Laura's data from 12 Feb.
states[17,1] <- 2
states[4, c(1,2)] <- c(1061, 299)
knitr::kable(states)
rowSums(states)
```



First step in the ancestral state reconstruction is changing these counts into proportions.  Three orders have no data so they will have equal probability of being in any state.
```{r}
for(i in 1:nrow(states)){
  if(sum(states[i, ]) == 0) states[i, ] <- c(1,1,1,1,1)
  states[i, ] <- states[i, ] / sum(states[i, ])
}
knitr::kable(states)
rm(i, temp.data, data)
#colnames(states) <- 1:5
```

Now lets try doing the ancestral state reconstruction using an ML approach.  I am going to use Liam's stochastic mapping approach becuase it will allow me to have uncertainty in the tip states.  This will allow me to use the information that we have for thousands of tips while still using our order backbone tree.

```{r}
# also we dont have data for archaegnatha and diplura so lets drop those from the analysis
new.tree <- drop.tip(new.tree, tip = c("diplura", "archaeognatha"))
states <- states[!row.names(states) %in% c("diplura", "archaeognatha"),]

reconstruction <- make.simmap(tree = new.tree, x = states, Q="mcmc",
                              model="ARD", pi="estimated", nsim=100)
#plotSimmap(reconstruction)
describe.simmap(reconstruction, plot=T)

```

This actually isn't bad as an unconstrained first approximation.  However, we would like to build several assumptions into our model that will reduce the number of parameters and make use of what we know from other sources of data.

1. transitions out of HD are likely ~ 0
2. transitions between between the two heterogametic systems would likely go through a 2 sex chromosome systems.  For example no transition from XY to ZO you would go from XY to ZW then from ZW to ZO.
3. we have very few transitions from between XY and ZW and few origins of HD so we will set XO and XY to ZW as a single rate, ZO and ZW to XY as a single rate, and XO, XY, ZO, ZW to HD as a single rate.

Here is the matrix form of those assumptions:

```{r}
my.mod <- matrix(c(0,4,0,0,0,
                   1,0,0,0,0,
                   2,2,0,5,0,
                   0,0,6,0,0,
                   3,3,3,3,0), 5)

```

Now lets try rerunning the reconstruction in this first one we will say that we know the root is either XY or XO.

```{r}
reconstruction <- make.simmap(tree = new.tree, x = states, Q="mcmc",
                              model=my.mod, pi=c(1,1,0,0,0), nsim=100)
#plotSimmap(reconstruction)
describe.simmap(reconstruction, plot=T)
```

Now we can compare how our root treatment effects our conclusions impact our reconstructed states. This time we will allow the roots to be estimated from the steady state distribution.

Now lets try rerunning the reconstruction in this first one we will say that we know the root is either XY or XO.

```{r}
reconstruction2 <- make.simmap(tree = new.tree, x = states, Q="mcmc",
                              model=my.mod, pi="estimated", nsim=1000, message=F)
```

now lets put these two constructions side by side and see what they look like

```{r}
par(mfcol=c(1,2))
describe.simmap(reconstruction, plot=T)
describe.simmap(reconstruction2, plot=T)

```

So we see that the extra constraint doesnt greatly change our ancestral states.  With that in mind lets focus on the second reconstruction which does not constrain the root and allows it to be estimated from the Q matrix.

Now let build a nice plot for the manuscript so we can talk about it.

```{r}
#first exctract out the values we want
for(i in 1:length(new.tree$tip.label)){
  name<- new.tree$tip.label[i]
  new.tree$tip.label[i] <- paste0(toupper(substr(name, 1, 1)), 
                                  substr(name, 2, nchar(name)))
}
hist <- summary(reconstruction2, plot=F)
plot(new.tree,cex=0.5, label.offset=.02)
#             HD           XO          XY       ZO         ZW
cols <- c("#ef6548", "#a1d99b", "#005a32", "#084594", "#9ecae1")
nodelabels(pie=hist$ace,piecol=cols,cex=0.5, width=2, height=1)
#             HD           XO          XY       ZO         ZW
cols <- c("#ef6548", "#005a32", "#a1d99b", "#084594"  ,"#9ecae1")
tiplabels(pie=states[,c(5, 1:4)],piecol=cols,cex=0.5)

#legend
text(x=rep(0.05, 5), y=c(30,29,28,27,26),
     label=c("XY", "XO", "ZW", "ZO", "HD/PGE"),
     cex=.7, pos=4)
points(x=rep(0.05, 5), y=c(30,29,28,27,26), cex=1.4,
       pch=16, col=cols[c(2:5, 1)])

foobar <- hist$ace
nodelabels()
#origin of insecs
hist$ace[rownames(hist$ace)==33,]
#origin of lep XO
hist$ace[rownames(hist$ace)==47,]
#origin of orthoptera
hist$ace[rownames(hist$ace)==54,]
#origin of blatodea
hist$ace[rownames(hist$ace)==57,]
#origin of mantodea
hist$ace[rownames(hist$ace)==56,]
```


