<!DOCTYPE html>
<html>
<title>Blackmon Lab</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
    
<meta name="keywords" content="evolution, coleoptera, genomics, karyotype, chromosome, academia, phylogeny">
<meta name="author" content="Heath Blackmon">    
    
<link rel="stylesheet" href="hlb.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<link rel="shortcut icon" href="beetle.png" />


<style>
body, h1,h2,h3,h4,h5,h6 {font-family: "Montserrat", sans-serif}
.w3-row-padding img {margin-bottom: 12px}
/* Set the width of the sidebar to 120px */
.w3-sidebar {width: 120px;background: #222;}
/* Add a left margin to the "page content" that matches the width of the sidebar (120px) */
#main {margin-left: 120px}
/* Remove margins from "page content" on small screens */
@media only screen and (max-width: 600px) {#main {margin-left: 0}}
</style>
<body class="w3-black">

<!-- Icon Bar (Sidebar - hidden on small screens) -->
<nav class="w3-sidebar w3-bar-block w3-small w3-hide-small w3-center">
  <!-- Avatar image in top left corner -->
    <img src="images/TAMU-white.png" alt="Texas A&M logo" style="width:100%">
    
  <a href="#" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>Home</p>
  </a>
  <a href="#research" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>Research</p>
  </a>
  <a href="#publications" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>Publications</p>
  </a>
  <a href="#resources" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>Resources</p>
  </a>
  <a href="#members" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>Members</p>
  </a>
  <a href="#news" class="w3-bar-item w3-button w3-padding-large w3-hover-black">
    <p>News</p>
  </a>
</nav>

<!-- Navbar on small screens (Hidden on medium and large screens) -->
<div class="w3-top w3-hide-large w3-hide-medium" id="myNavbar">
  <div class="w3-bar w3-black w3-opacity w3-hover-opacity-off w3-center w3-small">
    <a href="#" class="w3-bar-item w3-button" style="width:25% !important">Home</a>
    <a href="#research" class="w3-bar-item w3-button" style="width:25% !important">Research</a>
    <a href="#publications" class="w3-bar-item w3-button" style="width:25% !important">Publications</a>
    <a href="#resources" class="w3-bar-item w3-button" style="width:25% !important">Resources</a>
    <a href="#members" class="w3-bar-item w3-button" style="width:25% !important">Members</a>
    <a href="#news" class="w3-bar-item w3-button" style="width:25% !important">News</a>
  </div>
</div>
    
<!-- Page Content -->
    
<div class="w3-padding-large" id="main">
    
  <!-- Header/Home -->
  <header class="w3-container w3-center w3-black" id="home">
    <h1 class="w3-jumbo">Blackmon Lab</h1>
    <h3>Evolutionary Biology</h3>
    <img src="images/logo3.png" alt="blackmon lab logo" class="w3-image" width="25%">

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-15744659-3"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-15744659-3');
</script>

  </header>

  <!-- Research Section -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-64" id="research">
      <h2 class="w3-text-light-grey">Research</h2>
    <div class="w3-white"> <div class="w3-dark-grey" style="height:10px;width:0%"></div> </div>
      <p>The Blackmon Lab is in the <a href="http://www.bio.tamu.edu/" target="_blank" class="w3-hover-text-red">Biology Department at Texas A&amp;M University</a>. We have two broad areas of inquiry. The first is genome evolution, specifically sex chromosome and structural evolution. The second is the development of methods and databases that accelerate the analysis of data within a quantitative genetic or phylogenetic framework. To address these topics, we use a broad range of approaches including theoretical population genetics, bioinformatics, genomics, and molecular cytogenetics.  Although we have projects involving all types of organisms, we often study beetles, and we keep several species in the lab as model organisms.
    </p>
    <p class="w3-text-light-grey">Fragile Y Hypothesis</p>
      <p>Chromosomal sex determination is phylogenetically widespread, having arisen independently in many lineages. Decades of theoretical work provide predictions about sex chromosome differentiation that are well supported by observations in both XY and ZW systems. However, the phylogenetic scope of previous work gives us a limited understanding of the pace of sex chromosome gain and loss and why Y or W chromosomes are more often lost in some lineages than others, creating XO or ZO systems. Contrary to our initial expectations, we find that highly degenerated Y chromosomes of many members of the Coleoptera suborder Polyphaga are rarely lost and that cases of Y chromosome loss are strongly associated with chiasmatic segregation during male meiosis. We propose the <a href="blackmon.pubs/2014d.pdf" target="_blank" class="w3-hover-text-red">"Fragile Y Hypothesis"</a> that recurrent selection to reduce recombination between the X and Y chromosome leads to the evolution of a small pseudoautosomal region, which, in taxa that require XY chiasmata for proper segregation during meiosis, increases the probability of aneuploid gamete production, with Y chromosome loss. This hypothesis predicts that taxa that evolve achiasmatic segregation during male meiosis will rarely lose the Y chromosome.
      </p>
    <p class="w3-text-light-grey">Genetic Architecture</p>
      <p>The pace and direction of evolution are governed by the genetic architecture of trait variation. Evolutionary biologists have disagreed about whether genes can be considered to act in isolation, or in the context of their genetic background (Fisher Wright debate). Line cross analysis (LCA) estimates genetic architecture parameters conditional on the best model chosen from a vast model space using relatively few line means and ignores uncertainty in model choice. To address these issues, we introduced an <a href="blackmon.pubs/2016e.pdf" target="_blank" class="w3-hover-text-red">information theoretic approach</a> to LCA, which comprehensively assesses the potential model space, quantifies model selection uncertainty, and uses model weighted averaging to estimate composite genetic effects accurately. Using simulated data and previously published LCA studies we have shown the utility of our approach to define the components of complex genetic architectures. Our analysis of 20+ previously published datasets also shows that traditional approaches have underestimated the importance of epistasis.
      </p>
    <p class="w3-text-light-grey">Insect Karyotypes</p>
      <p>Insects exhibit variation in both chromosome number and sex chromosome systems. Insect karyotypes have been an important source of data for both taxonomic and basic evolutionary biology research. Unfortunately, this data has always been scattered among journals and dissertations that are not accessible without subscriptions. We created the Tree of Sex Database to make this data open and available to all researchers. We curate the insect portion of this database that currently has over 15,000 records and has proven to be a valuable resource to explore and test ideas about the evolution of high-level genome evolution.  Recently we <a href="blackmon.pubs/2017a.pdf" target="_blank" class="w3-hover-text-red">published a synthesis</a> of this vast dataset and uncovered many interesting patterns and avenues for future investigations. We also recently published a subset of this data as the <a href="http://www.uta.edu/karyodb/" target="_blank" class="w3-hover-text-red">Coleoptera Karyotype Database</a>.  Beetle karyotypes are particularly powerful since they normally include information about the meiotic behavior of sex chromosomes.
      </p>
    <p class="w3-text-light-grey">Chromosome Number</p>
      <p>It has long been thought that in Eusocial insect selection to increase genetic diversity within a colony should indirectly select for increases in the number of chromosomes. To test this long-standing hypothesis, <a href="blackmon.pubs/2015d.pdf" target="_blank" class="w3-hover-text-red">we investigated the relationship between eusociality and chromosome number across Hymenoptera</a>. We found that solitary and social Hymenoptera do not have significantly different numbers of chromosomes. However, we did find that chromosome number evolves more quickly in social than solitary Hymenoptera. It remains unclear whether variable selection pressure or drift are responsible for this difference.
      </p>
    <!-- End Research Section -->
  </div>
  <!-- Publication Section -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-16" id="publications">
    <h2 class="w3-text-light-grey">Publications</h2>
    <div class="w3-white"> <div class="w3-dark-grey" style="height:10px;width:0%"></div> </div><br><br>
      <a href="https://scholar.google.com/citations?user=ytApUl0AAAAJ&hl=en" target="_blank" class="w3-hover-text-red">Google Scholar</a>
      <p class="w3-text-light-grey">2017</p>
      <p >
          
          <span class="w3-text-white w3-hbold"> Blackmon H.,</span> Y. Brandvain. Long-term fragility of Y chromosomes is dominated by short-term resolution of sexual antagonism -  Genetics <a href="http://www.genetics.org/content/genetics/early/2017/10/10/genetics.117.300382.full.pdf" target="_blank" class="w3-hover-text-red">Early online</a>
          <br>
          <br>
          <span class="w3-text-white w3-hbold"> Blackmon H.,</span> Ross L, Bachtrog D. Sex determination, sex chromosomes and karyotype evolution in insects. – Journal of Heredity 108:1 78-93 – <em>recommended by Faculty of 1000. </em><a href="blackmon.pubs/2017a.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
          <br>
          <br>
          Adams R., D Schield, D. Card, <span class="w3-text-white w3-hbold">H. Blackmon</span>, and T. Castoe. GppFst: Genomic posterior predictive simulations of Fst and dxy for identifying outlier loci from population genomic data – Bioinformatics – doi:10.1093/bioinformatics/btw795. <a href="blackmon.pubs/2016b.pdf" target="_blank" class="w3-hover-text-red">PDF</a>

      </p>
      <p class="w3-text-light-grey">2016</p>
      <p>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J.P. Demuth.  An information-theoretic approach to estimating the composite genetic effects contributing to variation among generation means: moving beyond the joint-scaling test for line cross analysis. – Evolution 70:2 420-432. <a href="blackmon.pubs/2016e.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
          <br>
          <br>
          Asian Longhorn Beetle Consortium (67 Authors).  Genome of the Asian longhorned beetle (Anoplophora glabripennis), a globally significant invasive species, reveals key functional and evolutionary innovations at the beetle-plant interface. Genome Biology 17:1 227 - <em>Responsible for analysis of genome structure evolution. </em><a href="blackmon.pubs/2016d.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          Ross, L. and <span class="w3-text-white w3-hbold">H. Blackmon</span>.  Sex Determination. In R. Kliman (Ed.) Encyclopedia of Evolutionary Biology. 81-88 Elsevier Academic Press. doi:10.1016/B978-0-12-800049-6.00146-3. <a href="blackmon.pubs/2016a.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          Adams R.; <span class="w3-text-white w3-hbold">H. Blackmon</span>; J. Reyes-Velasco; D. Schield; D. Card; A. Andrew; N. Waynewood; T. Castoe. Microsatellite landscape evolutionary dynamics across 450 million years of vertebrate genome evolution. Genome 59:5, 295-310 - <em>Editor’s choice. </em><a href="blackmon.pubs/2016c.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
      </p>
      <p class="w3-text-light-grey">2015</p>
      <p>
          <span class="w3-text-white w3-hbold">Blackmon H.</span>, N. Hardy, L. Ross. The evolutionary dynamics of haplodiploidy: genome architecture and haploid viability. Evolution 69:11 2971-2978. <a href="blackmon.pubs/2015e.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J. P. Demuth. The fragile Y hypothesis: Y chromosome aneuploidy as a selective pressure in sex chromosome and meiotic mechanism evolution. Bioessays 37:9 942-950. <a href="blackmon.pubs/2015a.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J. P. Demuth. Coleoptera Karyotype Database. The Coleopterists Bulletin 69:1 174-175. <a href="blackmon.pubs/2015c.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          Ross, L., <span class="w3-text-white w3-hbold">H. Blackmon</span>, P. Lorite, V. Gokhman, and N. Hardy. Recombination, chromosome number and eusociality in the Hymenoptera. Journal of Evolutionary Biology 28:1 105-116. <a href="blackmon.pubs/2015d.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J. P. Demuth. Genomic origins of insect sex chromosomes. Current Opinion in Insect Science 7 45-50. – <em>recommended by Faculty of 1000. </em><a href="blackmon.pubs/2015b.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
      </p>
      <p class="w3-text-light-grey">2014</p>
      <p>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J. P. Demuth. Estimating tempo and mode of Y chromosome turnover: explaining Y chromosome loss with the fragile Y hypothesis. Genetics 197:2 561-572. <a href="blackmon.pubs/2014d.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> Coleoptera Karyotypes:  The evolution of sex chromosomes and chromosome number. Newsletter of the Ontario Entomological Society 19:2 19–21. <a href="blackmon.pubs/2014b.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          Streicher, J. W., T. J. Devitt, C. S. Goldberg, J. H. Malone, <span class="w3-text-white w3-hbold">H. Blackmon</span>, and M. K. Fujita. Diversification and asymmetrical gene flow across time and space: lineage sorting and hybridization in polytypic barking frogs. Molecular Ecology 23:13 3273-3291. <a href="blackmon.pubs/2014a.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
                        <br>
                        <br>
          Ashman T., D. Bachtrog, <span class="w3-text-white w3-hbold">H. Blackmon</span>, E.E. Goldberg, M.W. Hahn, M. Kirkpatrick, J. Kitano, J.E. Mank, I. Mayrose, R. Ming, S.P. Otto, C.L. Peichel, M.W. Pennell, N. Perrin, L. Ross, N. Valenzuela, and J.C. Vamosi. Tree of Sex: A database of sexual systems. Nature Scientific Data 1:140015. – <em>responsible for 11,526 invertebrate records and all figures. </em><a href="blackmon.pubs/2014c.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
      </p>
      <p class="w3-text-light-grey">2012</p>
      <p>
          <span class="w3-text-white w3-hbold">Blackmon H.</span> and J. P. Demuth. Ring Species and Speciation. Encyclopedia of Life Science. www.els.net. <a href="blackmon.pubs/2012a.pdf" target="_blank" class="w3-hover-text-red">PDF</a>
      </p>
      
    <!-- End Publication Section -->
  </div>

  <!-- Resources Section -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-16" id="resources">
    <h2 class="w3-text-light-grey">Resources</h2>
    <div class="w3-white"> <div class="w3-dark-grey" style="height:10px;width:0%"></div> </div>
      <br>
      <p class="w3-text-light-grey">Teaching</p>
      <p>
      <a href="http://coleoguy.github.io/expdes" target="_blank" class="w3-hover-text-red"><img src="images/expdes.png" alt="experimental design" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" ></a>
      This course is intended to provide a foundation in the proper design of scientific research projects in the field of biology. A wide range of biological experiments are covered, and each type of experiment is explored with an eye toward choosing the appropriate statistical technique for analysis. By the end of the course, successful students will be able to design biological studies that are statistically tractable and perform basic statistical analyses using the programming language R.
      </p><br>
      <br>
      <p>
      <img src="images/rseminar.jpg" alt="r in biology seminar" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      <a href="rseminar/index.html" target="_blank" class="w3-hover-text-red"><b>R for biologists seminar:</b></a> R is currently an essential tool for most evolutionary biologists.  This seminar includes tutorials and example datasets for many basic analyses in R that are commonly encountered in ecology and evolution.  The seminar starts off with the most basic steps (installation) so no prior experience is needed to use these exercises.</p><br><br>
      <p>
      <img src="images/foundations.jpg" alt="foundations of evolutionary biology" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      <a href="reading.group/index.html" target="_blank" class="w3-hover-text-red">Foundations of evolutionary biology:</a> An understanding of the foundational ideas that the modern synthesis of evolutionary biology is built on is essential for asking intelligent questions and understanding how your research fits into a larger understanding of life.  I offered a foundations of evolutionary biology seminar course for graduate students that provided a structured environment to discuss some of the most important papers in evolutionary biology. This website contains a schedule and PDF files for all of the readings in this course.</p>

      <p><a href="https://github.com/coleoguy" target="_blank" class="w3-hover-text-red"><b>GitHub software repository</b></a> I do most all of my development and analyses in public repositories.</p>
      
      <p class="w3-text-light-grey">R packages</p>
      <p>
      <img src="images/evobir.jpg" alt="R package evobir" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      The package <b><a href="https://github.com/coleoguy/evobir" target="_blank" class="w3-hover-text-red">evobiR</b></a> is a collection of R functions that I have developed for different projects. The functions in this package can be loosely organized into either analysis of empirical data, simulation functions, or those created primarily for didactic purposes. A <a href="http://coleoguy.github.io/evobir.html" target="_blank" class="w3-hover-text-red">vignette</a> demonstrating its use is available. Instructions for installing the latest version from github are available <a href="https://github.com/coleoguy/evobir" target="_blank" class="w3-hover-text-red">here</a>.</p><br><br>
      <p>
      <img src="images/saga.jpg" alt="R package saga" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      The package <a href="https://github.com/coleoguy/SAGA" target="_blank" class="w3-hover-text-red"><b>SAGA</b></a> is a collection functions to ease and hopefully improve the quality of line cross analysis of genetic architecture. The overall goal is to allow for an easy and straightforward implementation of model averaged analysis using AICc. A <a href="http://coleoguy.github.io/saga.pdf" target="_blank" class="w3-hover-text-red">vignette</a> demonstrating its use is available. Instructions for installing the latest version from github are available <a href="https://github.com/coleoguy/SAGA" target="_blank" class="w3-hover-text-red">here</a>.</p><br><br>
      <p class="w3-text-light-grey">Databases</p>
      <p>
      <img src="images/ckd.jpg" alt="coleoptera karyotype database" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      <a href="http://www.uta.edu/karyodb/" target="_blank" class="w3-hover-text-red">Coleoptera Karyotype Database:</a> Karyotypes are a highly variable and complex trait that offers an opportunity to detect changes in genome organization, uncover phylogenetic history, and distinguish cryptic species. The current database contains 4,861 Coleoptera karyotypes which we have compiled from 240 <a href="http://www.uta.edu/karyodb/refs.html" target="_blank" class="w3-hover-text-red">publications</a>. The <a href="http://www.uta.edu/karyodb/" target="_blank" class="w3-hover-text-red">Coleoptera Karyotype Database</a> allows users to select the records that they wish to retrieve based on either taxonomy or trait values and builds a customized table of the results in a new window.</p><br>
      <p>
      <img src="images/tos.jpg" alt="tree of sex database" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      The <a href="http://www.treeofsex.org" target="_blank" class="w3-hover-text-red">Tree of Sex Database:</a> this is a product of a working group sponsored by the National Evolutionary Synthesis Center which sought to compile all information on sex determination systems across the tree of life.  To date, we have collected over 30K records including more than 10K invertebrates.  Much like the Coleoptera Karyotype Database, our goal is to make this data widely available allowing researchers across the globe to use data that was previously difficult to collect. </p><br>
  <!-- End resources Section -->
  </div>
  <!-- members Section -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-16" id="members">
    <h2 class="w3-text-light-grey">Lab Members</h2>
    <div class="w3-white"> <div class="w3-dark-grey" style="height:10px;width:0%"></div> </div>
      
      
      <p class="w3-text-light-grey">Principal Investigator</p>
      <img src="images/heath_blackmon.jpg" alt="Heath Blackmon" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      <p>Heath Blackmon received his PhD in the Demuth lab at the University of Texas at Arlington in 2015 while studying the evolution of sex chromosomes and karyotypes in Coleoptera and invertebrates.  He then transitioned to a postdoc for two years at the University of Minnesota.  While there he collaborated with Emma Goldberg and Yaniv Brandvain on projects spanning population genetics and macroevolution.  Heath opened his own lab at Texas A&amp;M University in the fall of 2017.
          <br><br>
          <a href=blackmon.cv.pdf target="_blank" class="w3-hover-text-red">CV - January 2018</a><br><br>
      </p>
      
      
      <p class="w3-text-light-grey">Graduate Students</p>
      <p><img src="images/terrence.jpg" alt="terrence_sylvester" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      Terrence Sylvester
      <br>
      Terrence joined the lab as a biology Ph.D. student in 2017.  He is currently exploring the relationship between sexual systems and rates of chromosome evolution in Phasmatodea.  Terrence has broad interests in evolutionary biology.  He is working to develop a project that will allow him to combine field work with with his interest in the evolution of genome complexity and structure.
      <br><br><br><br><br><br>
      </p>

      <p class="w3-text-light-grey">Undergraduates</p>
      
      <p>Nathan Anderson
      <br>
      Nathan is a math major that joined the lab as an undergraduate researcher in 2017.  Nathan is working in R to develop a statistical test to detect when transitions in two discrete traits are found closer together than we would expect under a variety of possible null models.
      <br><br>
      </p>
          
      <p>Andrew Armstrong
      <br>
      Andrew is a math major that joined the lab as an undergraduate researcher in 2017.  Andrew is working to extend methods for the inference of genetic architecture by the analysis of line means.  This work will lead to a new release of our R package SAGA.
      <br><br>
      </p>
          
      <p><img src="images/paulina.png" alt="paulina_serra_rossi" class="w3-image" style="width:15%; float:left; padding: 0px 10px 10px 0px" >
      Paulina Serra Rossi
      <br>
      Paulina is a biology major that joined the lab as an undergraduate researcher in 2018.  Paulina is working on bioinformatics project to develop a python pipeline to infer changes in the size of the pseudoautosomal region of sex chromosomes by mapping short read data to assembled genomes.
      <br><br> <br>  
      </p>

      
      <p class="w3-text-light-grey"></p>
      <br>
      <p class="w3-text-light-grey">Join us</p>
      <p> I am currently recruiting a postdoc, graduate students, and undergraduates.  Interest in the evolution of sex chromosomes or genomic structure or more broadly trait evolution could make you a good fit in the lab.  While most of my work focuses on beetles motivated students wishing to work in other groups or on purely theoretical problems are also welcome.  Please read my <a href="expectations.html" target="_top"> expectations for lab members</a> and then <a href="mailto:coleoguy@gmail.com?Subject=joining the lab" target="_top">contact me</a> for more information.
      <br>
      <br>
      The application deadline for Texas A&amp;M is December 1st.  If you are interested you should email me so we can discuss your potential fit with the lab.  You can become part of the lab through three different degree programs.  There is the traditional <a href="http://www.bio.tamu.edu/index.php/grad/#apply" target="_blank" class="w3-hover-text-red">biology graduate program.</a> However, there are also two interdepartmental programs where you can earn a PhD in either <a href="http://eeb.tamu.edu/" target="_blank" class="w3-hover-text-red">EEB</a> or <a href="http://genetics.tamu.edu/" target="_blank" class="w3-hover-text-red">Genetics.</a>
      <br>
      </p>
      <p class="w3-text-light-grey">Collaborators</p>
      <a >Rich Adams</a><br>
      <a href="http://ib.berkeley.edu/labs/bachtrog/" target="_blank" class="w3-hover-text-red">Doris Bachtrog</a><br>
      <a href="https://brandvainlab.wordpress.com/" target="_blank" class="w3-hover-text-red">Yaniv Brandvain</a><br>
      <a href="http://www.snakegenomics.org/ToddCastoe/Home.html" target="_blank" class="w3-hover-text-red">Todd Castoe</a><br>
      <a href="http://www.uta.edu/faculty/demuth/" target="_blank" class="w3-hover-text-red">Jeff Demuth</a><br>
      <a href="http://www.uta.edu/faculty/mkfujita/" target="_blank" class="w3-hover-text-red">Matt Fujita</a><br>
      <a href="http://eeg.github.io/lab/home.html" target="_blank" class="w3-hover-text-red">Emma Goldberg</a><br>
      <a href="http://lauraross.bio.ed.ac.uk/" target="_blank" class="w3-hover-text-red">Laura Ross</a><br>
  <!-- End people Section -->
  </div>
  <!-- news section -->
  <div class="w3-content w3-justify w3-text-grey w3-padding-16" id="news">
    <h2 class="w3-text-light-grey">News</h2>
      
    <div class="w3-white"> <div class="w3-dark-grey" style="height:10px;width:0%"></div> </div>
      <ul type="disc">
         <li>
             Twitter News
              <br>
              <a class="twitter-timeline" data-width="50%" data-height="250" data-theme="dark" href="https://twitter.com/coleoguy">Tweets by coleoguy</a> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
              <br>
              <br>
              
          </li>
          <li>
              <strong>March 2018</strong>
              <br>
              Andrew and Nathan both did a great job presenting posters of their work in the lab at the Texas Genetics Conference.  Andrew presented a poster on our latest updates to the R package SAGA including some exciting extensions to traditional line cross analysis.  Nathan presented the work we have been doing to develop a new approach to detecting contingency in the evolution of binary traits.  Andrew was awarded best undergrad poster at the confernce - Congrats Andrew! 
              <img src="images/TGS.andrew.jpg" alt="Andrwe at the Texas Genetic Society" class="w3-image" style="width:50%; padding: 0px 10px 10px 0px" >
              <br>
              <br>

          </li>
          <li>
              <strong>October 2017</strong>
              <br>
              Read a bit more about the lab, my background, and plans for the future on the TAMU Biology website: <a href="http://www.bio.tamu.edu/index.php/2017/10/10/fs-blackmon/" target="_blank" class="w3-hover-text-red">Faculty spotlight</a>
              <br>
              <br>
               We now have a bright shiny new Invictus incubator from Genesee Scientific.  This large incubator is where we keep all of our permanent population lines.  Soon we should get our second incubator that will be used for experimental lines.
              <br>
              <img src="images/invictus.jpg" alt="Crysina" class="w3-image" style="width:50%; padding: 0px 10px 10px 0px" >
              <br>
              <br>

          </li>
          <li>
              <strong>August 2017</strong> 
              <br>
              The outcome of evolution is critically dependent on the nature of natural populations.  Are most species well characterized as large panmictic populations or do most species occur as many disjunct demes with limited gene flow?  This question is especially important for species like those in the genus <i>Chrysina</i> that are highly sought after by collectors.  If the population of one canyon is lost can they repopulate from another adjacent canyon or mountain range? This summer we collected all four species of north American <i>Chrysina</i>.  These beetles occur in West Texas, Arizona, and New Mexico.  Over the coming year, we will determine the degree to which gene flow occurs among the populations that we have sampled.
              <br><br>
              <br>
              <img src="images/chrysina.png" alt="Crysina" class="w3-image" style="width:75%; padding: 0px 10px 10px 0px" >
              <br>
              <br>
          </li>
          <li>
              <strong>June 2017</strong>
              The GSA did a nice write up on my plans for the lab on their <a href="http://genestogenomes.org/early-career-scientist-leadership-spotlight-heath-blackmon/" target="_blank" class="w3-hover-text-red">Genes to Genomes blog.</a>
          </li>
        <br>
      </ul>
      <br>
      <br>
      <iframe src="https://calendar.google.com/calendar/embed?mode=WEEK&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=coleoguy%40gmail.com&amp;color=%23333333&amp;ctz=America%2FChicago" style="border-width:0" width="100%" height="500" frameborder="0" scrolling="no"></iframe>
      
      
      <br>
      <br>
      <br>
      
      
                
                
      

      
      <p>  
      </p>
      <p> 
      </p>
      <!-- End field Section -->
    </div>
    <!-- Footer -->
<!-- END PAGE CONTENT -->
</div>
</body>
</html>