---
title: "Mutator Alleles - Evolvability"
author: "Heath Blackmon"
date: "2/13/2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Mutation rate should in general be as low as is physiologically possible.

Why?


# A simple popgen simulation. 

Lets imagine a populatin of haploid organisms with 100 genes. Each gene will start with a fitness of 1 and the total fitness of the organism is multiplicative. Lets set the first locus in the genome as a possible mutator allele (imagine a DNA repair gene). It has one allele that has normal mutation rate and another allele with high mutation rate. Lets say the normal rate of mutation is $10^{-5}$ and the mutator allele raises this to $10^{-2}$. The last thing we need to specify for our model is the distribution of fitnes effects for mutations. We will play with this a bit to see what impact it has. In this first simulation we will start out with a well adapted population and each generation one individual will be given the high mutation allele.

Here we build the base population
```{r}
# Lets represent our population as matrix each row is an individual each 
# column is a gene.
ne <- 1000
pop <- matrix(1, ne, 100)
# lets look at a little bit of the matrix
pop[1:5,1:12]
```

Now lets define two different DFEs.
```{r}
set.seed(1)
mut.bad <- abs(c(rnorm(500, sd=.5), rnorm(1000,sd=.1)))
mut.ok <- abs(c(rnorm(1000), sd=1,rnorm(500,sd=.1)))
par(mfcol=c(1,2))
plot(density(mut.bad), main="well adapted pop", xlab="fitness effect")
plot(density(mut.ok), main="poorly adapted pop", xlab="fitness effect")
```

Now lets code a population evolving

```{r}
# we will track the freq of the allele in this variable
freq <- c()
for(j in 1:1000){
  #print(j)
  # first we will make it so on average a mutator allele appears in 
  # one individual each generation
  mutant <- sample(1:1000, 1)
  pop[mutant, 1] <- 2
  # now the population produces offspring with a chance of passing on
  # mutations to them based on the allele they cary in position 
  # one of their genomes.

  # first low mutants
  low.muts <- which(pop[, 1] == 1)
  hits <- runif(min=0, max=1, n=length(low.muts)) < 10^-5
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.bad, 1)
    }
  }

  # now high mutants
  high.muts <- which(pop[, 1] == 2)
  hits <- runif(min=0, max=1, n=length(high.muts)) < 10^-3 * 100
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.bad, 1)
    }
  }

  # now calculate fitness
  fits <- apply(pop[,2:100], 1, prod)

  # now reproduce based on fitness
  pop <- pop[sample(1:ne, prob=fits, replace=T), ]
  
  # record the frequency of the mutator
  freq[j] <- (sum(pop[, 1] == 2))/ne
}
plot(freq, type = "l", 
     ylab = "frequency mutator allele",
     main = "simulation",
     ylim=c(0,1))
```

So we see that a mutator allele despite being introduced every generation it can't sweep. That was for a well adapted population. Now lets see how that compares to a poorly adapted population:


```{r}
# we will track the freq of the allele in this variable
pop <- matrix(1, ne, 100)
set.seed(2)
freq.unfitpop <- c()
for(j in 1:1000){
  #print(j)
  # first we will make it so on average a mutator allele appears in 
  # one individual each generation
  mutant <- sample(1:1000, 1)
  pop[mutant, 1] <- 2
  # now the population produces offspring with a chance of passing on
  # mutations to them based on the allele they cary in position 
  # one of their genomes.

  # first low mutants
  low.muts <- which(pop[, 1] == 1)
  hits <- runif(min=0, max=1, n=length(low.muts)) < 10^-5
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.ok, 1)
    }
  }

  # now high mutants
  high.muts <- which(pop[, 1] == 2)
  hits <- runif(min=0, max=1, n=length(high.muts)) < 10^-3 * 100
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.ok, 1)
    }
  }

  # now calculate fitness
  fits <- apply(pop[,2:100], 1, prod)

  # now reproduce based on fitness
  pop <- pop[sample(1:ne, prob=fits, replace=T), ]
  
  # record the frequency of the mutator
  freq.unfitpop[j] <- (sum(pop[, 1] == 2))/ne
}
plot(freq.unfitpop, type="l", 
     ylab="frequency mutator allele",
     main="simulation", ylim=c(0,1))
```


Now lets add to our simulation by also introducing a antimutator in every 10th generation




```{r}
# we will track the freq of the allele in this variable
pop <- matrix(1, ne, 100)
set.seed(2)
freq.unfitpop <- c()
for(j in 1:10000){
  #print(j)
  # first we will make it so on average a mutator allele appears in 
  # one individual each generation
  mutant <- sample(1:1000, 1)
  pop[mutant, 1] <- 2
  
  # we will also make an anti mutator appear each generation
  if(j %% 10 == 0){
    antimutant <- sample(1:1000, 1)
    pop[antimutant, 1] <- 1
  }
  # now the population produces offspring with a chance of passing on
  # mutations to them based on the allele they cary in position 
  # one of their genomes.

  # first low mutants
  low.muts <- which(pop[, 1] == 1)
  hits <- runif(min=0, max=1, n=length(low.muts)) < 10^-5
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.ok, 1)
    }
  }

  # now high mutants
  high.muts <- which(pop[, 1] == 2)
  hits <- runif(min=0, max=1, n=length(high.muts)) < 10^-3 * 100
  if(sum(hits) > 0){
    mutated <- which(hits)
    for(ii in 1:length(mutated)){
      pop[mutated[ii], sample(2:100, 1)] <- sample(mut.ok, 1)
    }
  }

  # now calculate fitness
  fits <- apply(pop[,2:100], 1, prod)

  # now reproduce based on fitness
  pop <- pop[sample(1:ne, prob=fits, replace=T), ]
  
  # record the frequency of the mutator
  freq.unfitpop[j] <- (sum(pop[, 1] == 2))/ne
}
plot(freq.unfitpop, type="l", 
     ylab="frequency mutator allele",
     main="simulation", ylim=c(0,1))
```
